addon/ActivityStreams.js

```diff
@@ -170,6 +176,67 @@ ActivityStreams.prototype = {
     }
   },
 
+  _processAndDispatchLinks(links, type) {
```

Comment:
There's a part I don't understand. Before this method used to call `this.send` how are the dispatched changes propagated to the content side now by `dispatch`?